src/components/join-flow/join-flow.jsx

```diff
@@ -1,19 +1,35 @@
+const bindAll = require('lodash.bindall');
```

Comment:
looks like we don't need bindall just yet

Reply:
we're using it below